packages/model-fragments/lib/fragments/ext.js

```diff
@@ -308,25 +309,24 @@ JSONSerializer.reopen({
 });
 
 // Retrieve or create a transform for the specific fragment type
-function getFragmentTransform(container, store, attributeType) {
-  var registry = container._registry || container.registry || container;
+function getFragmentTransform(appInstance, store, attributeType) {
```

Comment:
`getOwner` returns an `appInstance` that is like a mix of the container and the registry, so no need of accessing private apis.